qa/L0_shared_memory/shared_memory_test.py

```diff
@@ -301,7 +301,13 @@ def test_too_big_shm(self):
 
     def test_large_shm_register_offset(self):
         # Test for out of bounds read vulnerability when registering system shared memory with large offset
-        for platform in ["python", "onnx", "libtorch", "plan", "openvino"]:
+
+        platforms = (
+            ["python", "onnx", "libtorch", "plan", "openvino"]
+            if os.environ.get("BACKENDS") is None
+            else os.environ.get("BACKENDS").split(" ")
```

Comment:
Reliability can be improved by splitting on any number of white spaces (including tabs and new line chars)
```suggestion
            else os.environ.get("BACKENDS").split()
```

example:
```
>>> "1 2  3\n4".split(' ')
['1', '2', '', '3\n4']
>>> "1 2  3\n4".split()
['1', '2', '3', '4']
```